tests/test_adjoint_jacobian.py

```diff
@@ -377,6 +388,38 @@ def test_provide_starting_state(self, tol, dev_gpu):
 
         assert np.allclose(dM1, dM2, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize(
+        "old_obs",
+        [
+            qml.PauliX(0) @ qml.PauliZ(1),
+            qml.Hamiltonian([1.1], [qml.PauliZ(0)]),
+            qml.Hamiltonian([1.1, 2.2], [qml.PauliZ(0), qml.PauliZ(1)]),
+            qml.Hamiltonian([1.1, 2.2], [qml.PauliX(0), qml.PauliZ(0) @ qml.PauliX(1)]),
+        ],
+    )
+    def test_op_arithmetic_is_supported(self, old_obs, dev_gpu, tol):
```

Comment:
the parametrized `old_obs` is unused now, right?
```suggestion
    def test_op_arithmetic_is_supported(self, dev_gpu, tol):
```

Reply:
Yep, good catch. The overwriting definition can be removed. I'll rerun on a GPU node and push it up

Reply:
Done. Once the CI passes here (we may need to rerun them manually) we can merge.

Reply:
Nice!